packages/plugin-ext/src/main/browser/languages-main.ts

```diff
@@ -113,7 +113,7 @@ export class LanguagesMainImpl implements LanguagesMain {
                         suggestions: result.completions,
                         incomplete: result.incomplete,
                         // tslint:disable-next-line:no-any
-                        dispose: () => this.proxy.$releaseCompletionItems(handle, (<any>result)._id)
+                        dispose: () => this.proxy.$releaseCompletionItems(handle, (<any>result).id)
```

Comment:
Please get rid of the cast to <any>. If we're sure that result.id is set, we can write "result.id!". Generally, I don't really see why the id field is optional, but that goes beyond the scope of this PR.

Reply:
@evidolob can you explain why the id field is optional in `CompletionResultDto`
https://github.com/theia-ide/theia/blob/44a2ec7b3a8b31ebe8832c8ff00f3987910066fa/packages/plugin-ext/src/common/plugin-api-rpc-model.ts#L185-L196

Reply:
@kpge there are different classes extending `IdObject` and for some it can be optional. Since it is never a case `CompletionResultDto` then you can define more restrict type in the subclass, i.e. redeclare `id: string`.